web3/mutations/votes/commitVotes.ts

```diff
@@ -4,6 +4,28 @@ import { CommitVotes } from "types";
 export async function commitVotes({ voting, formattedVotes }: CommitVotes) {
   if (!formattedVotes.length) return;
 
+  if (formattedVotes.length === 1) {
```

Comment:
I did it like this at first, but then that would mean also having separate `useCommitVote` hooks for the mutations as well and it ended up adding like 5 files of duplicate code. 

same for below, you'd need to have a helper function with type information that can't be inferred like it is here (the types for the event data we get from the contract are quite unwieldy) so I went with this approach instead because it saved a lot of complexity.

so yeah I'm super open to a suggestion of how to do it like this without ballooning the complexity because that was my preferred approach, but I couldn't get it to work otherwise.